sycl/tools/CMakeLists.txt

```diff
@@ -4,7 +4,9 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 
 add_executable(get_device_count_by_type get_device_count_by_type.cpp)
 add_dependencies(get_device_count_by_type ocl-headers ocl-icd)
```

Comment:
```suggestion
```

Reply:
I cannot remember why this was added but I ran into situations where the use of the `OpenCL::Headers` target was not enough. I suggest to test this when removing it, e.g., by starting from a completely fresh repo with and without the OCL headers and Khronos ICD.